### PR TITLE
Feature/add contact view

### PR DESCRIPTION
**Pull request by ogonekmateusz**
image
image

### CODE REVIEW

File: src/assets/loho.png
Comment (michalzajac123) [inline src/assets/loho.png:1]: typo, should be `logo` instead of `loho`

File: src/components/HomeViewComponents/SectionHero.vue [hunk 1]
```diff
@@ -50,6 +51,24 @@ import 'bootstrap/dist/js/bootstrap.bundle';
 }
 
 .col-lg-10 {
-  font-size: 1.2rem !important; 
+  font-size: 1.2rem !important;
+  
+}
+.col-md-10{
```
Comment (michalzajac123) [inline src/components/HomeViewComponents/SectionHero.vue:57]: you should try to be consistent with the structure. 
You have `.name {` in one case, in second `.name{`
I think you should add space before `{`

File: src/assets/loho.png
Comment (ogonekmateusz) [reply src/assets/loho.png:1]: loho jest niewazne mordzia przeslalo sie bo chcialem favicon ale ono nie am znaczenia <3

File: src/components/HomeViewComponents/SectionHero.vue [hunk 2]
```diff
@@ -50,6 +51,24 @@ import 'bootstrap/dist/js/bootstrap.bundle';
...
+}
+img{
+  width: 120%;
+  height: 50vh;
+}
+@media (max-width: 986px) {
+  .col-md-10{
+    margin-top: 4rem;
+  }
+    img{
```
Comment (michalzajac123) [inline src/components/HomeViewComponents/SectionHero.vue:69]: is the formatting ok ?

**Review by michalzajac123 — APPROVED** [review]

**Pull request merged**